src/week1.fibonacci/fibonacci.js

```diff
@@ -0,0 +1,5 @@
+function solution(n) {
+  if(n == 0) return 0;
+  if(n == 1) return 1;
```

Comment:
```suggestion
  if (n <= 1) return n;
```

태성님 리뷰로 고친 후 런타임 문제가 해결되는지 궁금해요~